### DISCUSSION TITLE
article.summary as article.content[:50]

### DISCUSSION THREAD

**Discussion by ndeville** [getpelican/pelican#3052] [Q&A]
Not sure if my title is clear enough :)

I would like not to have to enter a Summary manually for each article, but instead the Summary (shown in article lists) to use the first X (say 50) characters of the article.

I have `SUMMARY_MAX_LENGTH = 50` in `pelicanconf.py` and tried to then use `article.content` in the Summary field in the HTML, to no avail.

What am I missing?

**Comment by avaris** [comment]
Should be `article.summary`, not `article.content`. By the way, `SUMMARY_MAX_LENGTH = 50` counts *words* not *characters*.

**Reply by ndeville** [reply]
I had tried that.... but now realise it did not work because I had an empty Summary field in my markdowns. All good, working now., thank you!
